### PR TITLE
Show all delivery attempts on RepeatRecord

### PR DESCRIPTION
**Pull request by dannyroberts**
You can now see all attempts at delivering a repeat record:

screen shot 2017-05-12 at 5 55 29 pm

You can also expand the [...] to see the error message (`failure_reason`):

screen shot 2017-05-12 at 5 55 41 pm

### CODE REVIEW

File: corehq/apps/repeaters/views.py [hunk 1]
```diff
@@ -75,7 +76,8 @@ def get(self, request, domain):
 
     def post(self, request, domain):
         # Retriggers a repeat record
-        record = self.get_record_or_404(request, domain)
+        record_id = request.POST.get('record_id')
+        record = self.get_record_or_404(request, domain, record_id)
```
Comment (proteusvacuum) [inline corehq/apps/repeaters/views.py]: cray. was this just never working?

Comment (dannyroberts) [inline corehq/apps/repeaters/views.py]: No I broke this last time i touched this :)

File: corehq/apps/repeaters/models.py [hunk 2]
```diff
@@ -639,7 +639,7 @@ def handle_failure(self, response):
         """Do something with the response if the repeater fails
         """
         return self._make_failure_attempt(
-            u'{}: {}. {}'.format(response.status_code, response.reason, getattr(response, 'content', None)),
+            u'{}: {}.\n{}'.format(response.status_code, response.reason, getattr(response, 'content', None)),
```
Comment (proteusvacuum) [inline corehq/apps/repeaters/models.py]: beautiful

**Review by proteusvacuum — APPROVED** [review]
This is huge.

**Comment by dannyroberts** [comment]
And yet surprisingly small :) Cool I'll merge after tests!

**Comment by benrudolph** [comment]
nice!

**Pull request merged**